src/xunit.console.netcore/Visitors/StandardOutputVisitor.cs

```diff
@@ -93,9 +96,28 @@ protected override bool Visit(ITestSkipped testSkipped)
 
         protected override bool Visit(ITestStarting testStarting)
         {
+            if (showProgress)
+            {
+                lock (consoleLock)
+                {
+                    Console.Error.WriteLine("   {0} [STARTING]", Escape(testStarting.Test.DisplayName));
+                }
```

Comment:
Don't know; I was just following the precedent used for some of the other ones, like SKIP.  I can certainly change to just use Console.WriteLine if you'd prefer.